db/schema.rb

```diff
@@ -20,6 +20,13 @@
     t.integer  "creator_id"
   end
 
+  create_table "options", force: :cascade do |t|
+    t.datetime "created_at",   null: false
+    t.datetime "updated_at",   null: false
+    t.text     "option_text"
+    t.integer  "option_total"
```

Comment:
So every user who bets on the same option (outcome) has to bet the same amount of money ?